src/osd/ReplicatedPG.cc

```diff
@@ -4351,8 +4352,8 @@ int ReplicatedPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
           if (result >= 0) {
 	    if (maybe_create_new_object(ctx)) {
               ctx->mod_desc.create();
+	      t->touch(soid);
```

Comment:
It looks to me like you don't need this.  It also requires that you add af396b5aa77cbbf603211e1a08856a112ab43a6c to compensate.  This is a backport, it's important to be extremely careful about what we include and why.